docker-compose-perf.yml

```diff
@@ -789,6 +827,32 @@ services:
     profiles:
       - ttk-provisioning-transfers
 
+  ttk-provisioning-quotes:
+    image: mojaloop/ml-testing-toolkit-client-lib:${TEST_TTK_CLI_VERSION}
+    env_file:
+      - .env
+      - perf.env
+    volumes:
+      - "./docker/ml-testing-toolkit/test-cases/collections:/opt/app/collections"
+      - "./docker/ml-testing-toolkit/test-cases/environments:/opt/app/environments"
+      - "./reports:/opt/app/reports"
+    depends_on:
+      mojaloop-testing-toolkit:
+        condition: service_healthy
+      central-ledger:
+        condition: service_healthy
+      central-handler-admin:
```

Comment:
Why central-handler-admin is a dependency here?

Reply:
copy paste. removed.